code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,6 +15,11 @@
 	if(istype(holder))
 		insert(holder)
 
+/obj/item/organ/internal/Destroy()
```

Comment:
Ahh, it's this one that needs to be removed. I'm an idiot and confused you: I meant the one commetned on needs to be removed becuase of the one I linked to, not the one I linked to. My mistake.